modules/lang-painless/src/test/java/org/elasticsearch/painless/ImplementInterfacesTests.java

```diff
@@ -35,9 +36,52 @@
  * Tests for Painless implementing different interfaces.
  */
 public class ImplementInterfacesTests extends ScriptTestCase {
```

Comment:
We should rename this, since it is no longer an interface. Perhaps `BaseClassTests`?